### ISSUE TITLE
Proper ordering of <h> tags in the pages

### ISSUE THREAD

**Issue by AyanChoudhary** [creativecommons/cccatalog-frontend#1118]
## Problem

We need to have a proper ordering of the ```<h>``` tags. Currently, we have an ```<h5>``` tag in the footer and an ```<h1>``` or ```<h2>``` tag in the page. Ideally to satisfy accessibility requirements we need to have the ```<h>``` tags in a definite order i.e. an ```<h5>``` tag should have preceding ```<h1> ... <h4>```tags.

## Additional context

This is actually reducing our accessibility scores by quite a bit.

## Implementation

- [x] I would be interested in implementing this feature.

**Comment by Aakash2408** [comment]
Hey @AyanChoudhary  , @kgodey  can you explain a bit more what changes we need in <h> tags?

**Comment by AyanChoudhary** [comment]
@Aakash2408 The a11y requirements state that we need to have a proper hierarchy of h tags, but most of our pages only have an ```<h1>``` or at most have a level hierarchy till ```<h2>``` tags. We currently have an ```<h5>``` tag in our footer [here](https://github.com/creativecommons/cccatalog-frontend/blob/0d7b5926db07dafd42ad0a95c3ae43b5b6957ff2/src/components/FooterSection.vue#L150).

What we need to do is change the ```<h5>``` to a ```div```/```span``` while maintaining the ```<h5>``` styling. Maybe this [recent change](https://github.com/creativecommons/vocabulary/pull/553) in vocabulary would help

**Comment by Aakash2408** [comment]
@AyanChoudhary  so we need to change every <h> tag into any div or span but keeping the styling like font size keeping the same?

**Comment by AyanChoudhary** [comment]
No, just the <h5> in the footer needs to be changed.

**Comment by Aakash2408** [comment]
ok ! @AyanChoudhary  Can I take this?

**Comment by AyanChoudhary** [comment]
Sure! Go ahead 👍

**Comment by Aakash2408** [comment]
@AyanChoudhary  can you kindly tell me the location of code  I mean in which branch it is-- develop or master and where?

**Comment by rochisha0** [comment]
@Aakash2408 `develop` is the main branch.

**Comment by Aakash2408** [comment]
> No, just the `<h5>` in the footer needs to be changed.
  https://github.com/creativecommons/cccatalog-frontend/blob/develop/src/components/FooterSection.vue Line 150  .It looks this is the only h5  in the footer

**Comment by Aakash2408** [comment]
Hey @rochisha0  @AyanChoudhary   @kgodey 
When I try to run the project locally after npm install  and when I run the command npm run dev 
![Screenshot from 2020-08-18 15-04-57](https://user-images.githubusercontent.com/47134609/90497135-5ce96400-e164-11ea-916c-87371741fa9e.png)
Do anyone have any idea what is happening?

**Comment by rochisha0** [comment]
Try merging it with the upstream and repeat the process @Aakash2408

**Comment by AyanChoudhary** [comment]
@Aakash2408 try removing node_modules and re run `npm install` that should fix it

**Comment by Aakash2408** [comment]
@AyanChoudhary  @kgodey  kindly review the Pull Request
